src/partials.js

```diff
@@ -157,11 +157,19 @@ export const AddToGitIgnore = ({ path = "/screenshots" }) => (
   </p>
 );
 
+export const NoNeedUpdateCI = () => (
+  <p>
+    <span className="font-medium">Note:</span> No need to add{" "}
+    <code>@argos-ci/cli</code> in your CI/CD pipeline. Screenshots and traces
+    are automatically uploaded to Argos when your tests run on CI.
```

Comment:
```suggestion
    <span className="font-medium">Note:</span> You don't need to install and run `@argos-ci/cli`. Screenshots and traces
    are automatically uploaded to Argos when your tests run on CI.
```

Reply:
A markdown partial would be better for this.

https://docusaurus.io/docs/markdown-features/react#importing-markdown